src/components/Homepage/OurWork.js

```diff
@@ -96,7 +97,24 @@ const AnimatedLink = styled(Link)`
       transform: scale(0.97);
     }
   }
+
+  ${breakpoint('tablet')`
+    margin-top: ${({ theme }) => theme.space[5]};
+  `}
+`
+
+const CaseStudyImageWrapper = styled.div`
+  background-color: #${({ backgroundColor }) => backgroundColor};
+
+  > * {
```

Comment:
Why is this necessary?

Reply:
Ah yes, should have put a comment in here. this is because at some sizes the gap between the image and the header section was big enough (1 pixel) to see the grey background of the section it's on. This just prevents that

Reply:
Can we put it on the child itself?